docs/index.md

```diff
@@ -0,0 +1,117 @@
+<div style="display: flex; align-items: center; gap: 20px;">
```

Comment:
Not ideal that this is duplicated, is there any way to generate this when building the docs? or, worst case, test that it's in-sync?

Reply:
What do you mean is duplicated?

Reply:
As in, a lot of this seems to duplicate the top-level `README.md`, e.g. the code sample, the presentations list, the blogs list

Reply:
Right, I think it's fine for now and we can build it from source if we find we keep changing the same parts in both (so really only if someone is doing presentations and blogs to add them, the example isn't going to change for now).